packages/playground/src/components/app-root/app-root.tsx

```diff
@@ -1,33 +1,53 @@
 import { Component, State } from "@stencil/core";
-
-// @ts-ignore
-// Needed due to https://github.com/ionic-team/stencil-router/issues/62
 import { MatchResults } from "@stencil/router";
 
+import AccountTunnel, { State as AccountState } from "../../data/account";
+import NetworkTunnel, { State as NetworkState } from "../../data/network";
+
 @Component({
   tag: "app-root",
   styleUrl: "app-root.scss",
   shadow: true
 })
 export class AppRoot {
+  @State() accountState: AccountState = {};
+  @State() networkState: NetworkState = {
```

Comment:
What's the representation of a network state supposed to be atm?

Reply:
@snario Currently it is hardcoded to be "Lorem".
This will need to be inserted according to the actual network.